pkg/errors/factory_test.go

```diff
@@ -154,14 +165,49 @@ func TestIsForbiddenError(t *testing.T) {
 		},
 	}
 	for _, tc := range cases {
-		tc := tc
 		t.Run(tc.Name, func(t *testing.T) {
 			t.Parallel()
-			assert.Equal(t, IsForbiddenError(tc.Error), tc.Forbidden)
+			assert.Equal(t, IsForbiddenError(GetError(tc.Error)), tc.Forbidden)
 		})
 	}
 }
 
 func TestNilError(t *testing.T) {
 	assert.NilError(t, GetError(nil), nil)
 }
+
+func TestIsInternalError(t *testing.T) {
+	cases := []struct {
+		Name     string
+		Error    error
+		Internal bool
+	}{
+		{
```

Comment:
Let's add `api_errors.NewTimeoutError` just to be sure.

Reply:
It can be a separate test function or maybe a mix of different `StatusError` kinds.